docs/plugins.md

```diff
@@ -0,0 +1,229 @@
+## Plugin Support
+
+You can customize the linter with rules that are more specific to your use case with the `ember-template-lint` plugin system.
+
+Plugins can define new rules and set up default configurations that can be extended.
+
+### Defining plugin objects
+
+Each plugin object can include these properties.
+
+* `name` -- `string` (required)
+
+  The name of the plugin. Will be used to namespace any configuration objects.
+
+* `rules` -- `Object`
+
+  Object that defines new rules.
+  Each key represents the name of the rule that is defined.
+  Each value should be a string that represents a relative file path to the rule definition. See [Rule APIs](#rule-apis) for more detail.
```

Comment:
The values are actual "Rules" objects (not a string).  Commonly it is done as in the example below:

```js
rules: {
  'some-rule-name': require('./rules/some-rule-name')
}
```